tests/pyscript/bar1.py

```diff
@@ -1,3 +0,0 @@
-# flake8: noqa F821
```

Comment:
Is deleting the majority of our **pyscript** unit tests the right thing to do?  Or would it be better to convert them to using the other syntax given that is the only supported one now?

Reply:
The deleted tests only exercised the `ArgparseFunctor` stuff.

The remaining interface to `pyscript` is simple because it does no translation before sending the command to our parser. It is pretty well covered by existing unit tests.

Reply:
Maybe take a quick look at code coverage within ``do_pyscript`` and ``pyscript_bridge``.  But I think your current approach looks OK overall.